charts/gardener/charts/application/templates/apiservice-v1beta1-core-gardener-cloud.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.global.apiserver.enabled }}
+apiVersion: {{ include "apiserviceversion" . }}
+kind: APIService
+metadata:
+  name: v1beta1.core.gardener.cloud
+  labels:
+    app: gardener
+    role: apiserver
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  insecureSkipTLSVerify: {{ .Values.global.apiserver.insecureSkipTLSVerify }}
+  {{- if not .Values.global.apiserver.insecureSkipTLSVerify }}
+  caBundle: {{ required ".Values.global.apiserver.caBundle is required" (b64enc .Values.global.apiserver.caBundle) }}
+  {{- end }}
+  group: core.gardener.cloud
+  version: v1beta1
+  groupPriorityMinimum: 9
+  versionPriority: 9
```

Comment:
Don't you think it's already good to give it a higher prio than v1alpha1?

Reply:
You're right, thanks, done.